packages/big-design/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -330,8 +330,6 @@ export const MultiSelect = typedMemo(
                 !isOpen && openMenu();
               },
               onFocus: (event) => {
-                !isOpen && openMenu();
-
```

Comment:
❤️

Reply:
Thanks for the comment i completely forgot `MultiSelect` was a thing HAHA 😅 😅